src/libraries/System.Runtime/tests/TrimmingTests/StackFrameHelperTest.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+using System.Diagnostics;
+
+class Program
+{
+    static int Main(string[] args)
+    {
+        // StackTrace constructor calls CaptureStackTrace method which in turn will call
+        // StackFrameHelper.InitializeSourceInfo which has annotations as it will require
```

Comment:
`InitializeSourceInfo` has a big try/catch and ignores any exceptions. You also need to validate the result to tell whether `InitializeSourceInfo` worked properly.